src/_csrc/pycabcrypt.h

```diff
@@ -14,15 +14,20 @@ typedef unsigned long long uint64_t;
 typedef uint64_t u_int64_t;
 #define snprintf _snprintf
 #define __attribute__(unused)
-#define __BEGIN_DECLS
-#define __END_DECLS
 #else
 #include <stdint.h>
 #endif
+#if !defined(__BEGIN_DECLS)
+#define __BEGIN_DECLS
+#endif
+#if !defined(__END_DECLS)
+#define __END_DECLS
+#endif
 
 #define explicit_bzero(s,n) memset(s, 0, n)
 #define DEF_WEAK(f)
 
 int bcrypt_hashpass(const char *key, const char *salt, char *encrypted, size_t encryptedlen);
 int encode_base64(char *, const u_int8_t *, size_t);
 int timingsafe_bcmp(const void *b1, const void *b2, size_t n);
+int bcrypt_pbkdf(const char *pass, size_t passlen, const uint8_t *salt, size_t saltlen, uint8_t *key, size_t keylen, unsigned int rounds);
```

Comment:
Wait, did we just cancel the old PR that previously exported this? I thought we already exported this.